src/Todo/Bundle/ListBundle/Controller/Api/ListController.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Todo\Bundle\ListBundle\Controller\Api;
+
+use Doctrine\ORM\Mapping as ORM;
+use FOS\RestBundle\Controller\FOSRestController;
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Todo\Bundle\ListBundle\Entity\ListItem;
+use FOS\RestBundle\Controller\Annotations as Rest;
+use Todo\Bundle\ListBundle\Form\ListType;
+use Todo\Bundle\UserBundle\Entity\User;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="list_controller")
+ */
+class ListController extends FOSRestController
+{
+    /**
+     * @Rest\Get("/api/list")
+     * @ApiDoc(
+     *  section="List",
+     *  description="Returns User Lists"
+     * )
+     * @param User $user
+     */
+    public function getListsAction(User $user)
+    {
+        if (!$this->getUser()) {
+            throw new AccessDeniedHttpException("Access Denied");
+        }
+        $em = $this->getDoctrine()->getManager();
+
+        $lists = $em->getRepository('TodoBundleListBundle:ListItem')->getUserLists($user);
+
+       return new JsonResponse($lists, Response::HTTP_ACCEPTED);
+    }
+
+    /**
+     * Get Items of Specific List.
+     *
+     * @param ListItem $listItem
+     * @return JsonResponse
+     *
+     * @Rest\Get("/api/list/{id}")
```

Comment:
`@Rest\Get("/api/list/{id}/items")` 
One list has many items :)

Reply:
all the routes was from Doc of interview task , but no problem i will update them :D